uber/models.py

```diff
@@ -890,7 +1021,7 @@ def model_mixin(cls, model):
         for name in dir(model):
             if not name.startswith('_'):
                 attr = getattr(model, name)
-                if hasattr('target', '__table__') and name in target.__table__.c:
+                if hasattr(target, '__table__') and name in target.__table__.c:
```

Comment:
Huh, that's... interesting. Our model mixins have been working fine, or so I thought.

Considering that this is unrelated to the PR and the results are unclear, do you think it's a good idea to separate this into another PR? That way if something goes haywire it's easy to revert.